website/docusaurus.config.js

```diff
@@ -382,6 +382,10 @@ module.exports = {
                         from: '/sdks/proxy-vue',
                         to: '/reference/sdks/vue',
                     },
+                    {
+                        from: '/sdks/proxy-flutter',
+                        to: '/reference/sdks/flutter-proxy',
+                    },
```

Comment:
I don't think we need to add a redirect here. The old version never existed, right?

```suggestion
```